app/assets/javascripts/application.js

```diff
@@ -8,7 +8,6 @@
 //= require govuk_publishing_components/components/step-by-step-nav
 
 //= require support
-//= require browse-columns
```

Comment:
It'd be really interesting to see what effect the removal of the CSS and JavaScript has on page weight - can you give me a shout when this gets deployed to production so I can add a note to Speedcurve?

Reply:
Yes, definitely, will do 👍